src/lib/cozy-client/DataAccessFacade.js

```diff
@@ -18,16 +18,22 @@ export default class DataAccessFacade {
     this.pouchAdapter = new PouchdbAdapter()
   }
 
-  setup(cozyUrl, options) {
-    this.url = cozyUrl
-    const { offline, ...rest } = options
+  setup(cozyURL, options) {
+    this.url = cozyURL
+    const { offline } = options
+
     // TODO: For now we let cozy-client-js handle offline for files so that we don't break cozy-drive
-    const config =
+    const hasFilesDoctype =
       offline &&
       offline.doctypes &&
       offline.doctypes.indexOf(FILES_DOCTYPE) !== -1
```

Comment:
Tip: In Collect, we are no using [Array.prototype.includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) it's far better and easier to read than this good old `indexOf`.